HW1/P6/P6.py

```diff
@@ -0,0 +1,63 @@
+import findspark
+findspark.init('/Users/george/Documents/spark-1.5.0')
+import pyspark
+import random
+
+sc = pyspark.SparkContext()
+
+# filter out upper case and all digits
+def filter_word(word):
+    return not word.isdigit() and not word.isupper()
```

Comment:
I think that word.isupper() will also filter out "UPPERCASE." 

For example: "HELLO.".isupper() returns True.